setup.py

```diff
@@ -28,7 +28,7 @@
 
 # Constants #########################################################
 
-VERSION = '4.0.6'
+VERSION = '3.4.23'
```

Comment:
@mudassir-hafeez Why is this downgraded?

Reply:
@aayushagra thank you for taking my attention towards it. Actually, if you see releases, we were using two different versions of releases. one is for Ironwood Open edX releases and other is for normal releases. After Ironwood upgrade, releases should be as they were before. @xitij2000 can you please comment on that what should be the right release version from now.